cmake/find/poco.cmake

```diff
@@ -8,6 +8,14 @@ if (NOT EXISTS "${ClickHouse_SOURCE_DIR}/contrib/poco/CMakeLists.txt")
    set (MISSING_INTERNAL_POCO_LIBRARY 1)
 endif ()
 
+if (NOT ENABLE_LIBRARIES)
+    set (ENABLE_POCO_NETSSL ${ENABLE_LIBRARIES} CACHE BOOL "")
```

Comment:
This doesn't work for CMake ≤ 3.13 - it doesn't respect variables before options with a same name.

Reply:
Even with CACHE BOOL "" ?